docs/specs/multi-registry/multi_registry.md

```diff
@@ -0,0 +1,501 @@
+
+Sections such as [Registry Resolution](#registry-resolution), [Registry Packaging](#registry-packaging), and 
+[Registry Publication](#registry-publication) provide more details on how the `weaver_registry.yaml` file
+is used in these processes.
+
+Open Questions:
+- Should we follow SemVer 2 for registry versions? It seems advisable, as Weaver can detect breaking changes. However,
+  the resolution process doesn't rely on SemVer 2 in an important way.
+- How do we retrieve a published registry URL from a schema URL? Perhaps a new field in the schema URL file could
+  point to the published registry URL, possibly named `registry_url`.
+
+### Semantic Convention File Structure
+
+The current [JSON Schema](https://github.com/open-telemetry/weaver/blob/main/schemas/semconv.schema.json) that
+describes the files of the semantic convention registry remains mostly unchanged. However, a few minor updates are
+required:
+
+- The ability to reference a group must be supported, allowing a registry to extend or override fields of an
+  imported group.
+- The structure of a reference must be enhanced to disambiguate attribute or signal references from other registries
+  when necessary (more details in the [Registry Resolution](#registry-resolution) section).
+
+Currently, a reference follows dot notation (e.g., `ref: client.address`). When there is no ambiguity or conflict
+detected by Weaver, this notation is sufficient. However, when a conflict occurs, the user must disambiguate the
+reference by prefixing it with the registry name (e.g., `ref: otel:client.address`). The prefix corresponds
+to the registry name as defined in the `weaver_registry.yaml` file under the `dependencies` section. A colon is used
+as a separator between the name and the attribute or group name. References to groups or attributes defined in the
+local registry are never prefixed.
+
+In a group, the `extends` section can now reference an imported group using this new reference syntax. Similarly, the
+`attributes` section of a group can reference an imported attribute.
+
+In the current semantic conventions specification, referencing a group is currently unsupported. Uniqueness within
+groups is scoped by the type of group. It is entirely possible to have an event and a metric identified by the same ID.
+Therefore, the following approach is proposed to reference a group:
+
+```yaml
+groups:
+  - metric_ref: <reference_to_a_metric_group>
+  - event_ref: <reference_to_an_event_group>
+  - span_ref: <reference_to_a_span_group>
+  - ...
+```
+
+Similar to attribute references, group references cannot override the type, stability, or deprecation status of the
+imported group.
+
+References to experimental entities across registries are allowed under certain conditions:
+
+- An entity referencing an experimental entity must also be marked as experimental.
+- The flag `allow_experimental_ref` in the `weaver_registry.yaml` file must be set to `true`.
```

Comment:
`allow_experimental_ref` - could we instead recommend (and include) a set of default policies to run when referencing things? I assume there will be tons of other policy-related things (charset, naming, deprecation/removal/back-compat, etc) that depend on the project needs and it'd be nice to make this configurable/expandable via a policy set rather than config options in the manifest